03:07:02 PM: DEBUG [loggerBase.py:14] Mensaje a nivel debug
03:07:02 PM: INFO [loggerBase.py:15] Mensaje a nivel info
03:07:02 PM: WARNING [loggerBase.py:16] Mensaje a nivel warining
03:07:02 PM: ERROR [loggerBase.py:17] Mensaje a nivel error
03:07:02 PM: CRITICAL [loggerBase.py:18] Mensaje a nivel critical
03:07:08 PM: DEBUG [loggerBase.py:14] Mensaje a nivel debug
03:07:08 PM: INFO [loggerBase.py:15] Mensaje a nivel info
03:07:08 PM: WARNING [loggerBase.py:16] Mensaje a nivel warining
03:07:08 PM: ERROR [loggerBase.py:17] Mensaje a nivel error
03:07:08 PM: CRITICAL [loggerBase.py:18] Mensaje a nivel critical
12:32:45 PM: DEBUG [persona.py:52] 
        ID: Joan, Nombre: Herrera, 
        Apellido: Joanherrera@gmail.com, Correo: None
        
12:34:20 PM: DEBUG [persona.py:52] 
        ID: 1, Nombre: Joan, 
        Apellido: Herrera, Correo: Joanherrera@gmail.com
        
12:37:29 PM: DEBUG [persona.py:52] 
        ID: 1, Nombre: Joan, 
        Apellido: Herrera, Correo: Joanherrera@gmail.com
        
12:37:29 PM: DEBUG [persona.py:55] 
        ID: None, Nombre: Alexis, 
        Apellido: Vilte, Correo: Alexisvilte@gmail.com
        
12:37:29 PM: DEBUG [persona.py:58] 
        ID: 1, Nombre: None, 
        Apellido: None, Correo: None
        
12:38:36 PM: DEBUG [persona.py:52] 
        ID: Joan, Nombre: Herrera, 
        Apellido: Joanherrera@gmail.com, Correo: None
        
12:38:36 PM: DEBUG [persona.py:55] 
        ID: None, Nombre: Alexis, 
        Apellido: Vilte, Correo: Alexisvilte@gmail.com
        
12:38:36 PM: DEBUG [persona.py:58] 
        ID: 1, Nombre: None, 
        Apellido: None, Correo: None
        
12:38:40 PM: DEBUG [persona.py:52] 
        ID: Joan, Nombre: Herrera, 
        Apellido: Joanherrera@gmail.com, Correo: None
        
12:38:40 PM: DEBUG [persona.py:55] 
        ID: None, Nombre: Alexis, 
        Apellido: Vilte, Correo: Alexisvilte@gmail.com
        
12:38:40 PM: DEBUG [persona.py:58] 
        ID: 1, Nombre: None, 
        Apellido: None, Correo: None
        
12:38:50 PM: DEBUG [persona.py:52] 
        ID: 1, Nombre: Joan, 
        Apellido: Herrera, Correo: Joanherrera@gmail.com
        
12:38:50 PM: DEBUG [persona.py:55] 
        ID: None, Nombre: Alexis, 
        Apellido: Vilte, Correo: Alexisvilte@gmail.com
        
12:38:50 PM: DEBUG [persona.py:58] 
        ID: 1, Nombre: None, 
        Apellido: None, Correo: None
        
12:52:45 PM: DEBUG [loggerBase.py:14] Mensaje a nivel debug
12:52:45 PM: INFO [loggerBase.py:15] Mensaje a nivel info
12:52:45 PM: WARNING [loggerBase.py:16] Mensaje a nivel warining
12:52:45 PM: ERROR [loggerBase.py:17] Mensaje a nivel error
12:52:45 PM: CRITICAL [loggerBase.py:18] Mensaje a nivel critical
12:52:50 PM: DEBUG [persona.py:52] 
        ID: 1, Nombre: Joan, 
        Apellido: Herrera, Correo: Joanherrera@gmail.com
        
12:52:50 PM: DEBUG [persona.py:55] 
        ID: None, Nombre: Alexis, 
        Apellido: Vilte, Correo: Alexisvilte@gmail.com
        
12:52:50 PM: DEBUG [persona.py:58] 
        ID: 1, Nombre: None, 
        Apellido: None, Correo: None
        
01:48:03 PM: ERROR [conexion.py:28] Ocurrio una excepcion al obtener la conexion invalid dsn: invalid connection option "dbport"

01:51:25 PM: ERROR [conexion.py:28] Ocurrio una excepcion al obtener la conexion invalid dsn: invalid connection option "dbport"

01:54:27 PM: ERROR [conexion.py:28] Ocurrio una excepcion al obtener la conexion invalid dsn: invalid connection option "dbport"

01:54:35 PM: ERROR [conexion.py:28] Ocurrio una excepcion al obtener la conexion invalid dsn: invalid connection option "dbport"

01:57:01 PM: ERROR [conexion.py:28] Ocurrio una excepcion al obtener la conexion connection to server at "localhost" (127.0.0.1), port 5432 failed: FATAL:  database "testdb" does not exist

01:57:15 PM: ERROR [conexion.py:28] Ocurrio una excepcion al obtener la conexion invalid dsn: invalid connection option "DBport"

09:51:01 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7ff301109d50; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:51:11 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7fae2c0909c0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:51:15 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7f3dbaffb9c0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:51:49 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7f648c1ac9c0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:53:29 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7ff4faf22c20; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:54:03 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7ff65ccd3c20; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:54:17 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7f5a73b1baf0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:55:01 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7fa6141d69c0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
09:55:57 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7ffb693d7e80; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
10:48:37 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7fd8602449c0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
10:48:44 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7fc97b4b43d0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
10:50:02 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7f5cfc18fc20; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
04:08:44 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7fe890f6b3d0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
04:08:44 PM: DEBUG [conexion.py:38] Se abrio correctamente el cursor: <cursor object at 0x7fe88fc4d4f0; closed: 0>
04:08:48 PM: DEBUG [conexion.py:25] Conexion exitosa <connection object at 0x7f7e92af53d0; dsn: 'user=postgres password=xxx dbname=postgres host=localhost port=5432', closed: 0>
04:08:48 PM: DEBUG [conexion.py:38] Se abrio correctamente el cursor: <cursor object at 0x7f7e9183e4f0; closed: 0>
